stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -208,7 +208,7 @@ end
 
 function show(io::IO, M::Bidiagonal)
     # TODO: make this readable and one-line
-    println(io, summary(M), ":")
+    summary(io, M); println(io, ":")
```

Comment:
doesn't seem like these need to share a line

Reply:
Right, they don't *need* to. But nevertheless they do.